plotly/tools.py

```diff
@@ -1670,6 +1722,321 @@ def create_streamline(x, y, u, v,
                              mode='lines', **kwargs)
         return streamline
 
+    @staticmethod
+    def create_ohlc(open, high, low, close, direction, **kwargs):
+
+        """
+        Returns data for an ohlc chart
```

Comment:
ah, good call. yeah we should be careful about using "data", it's so easy to throw around, but probably super confusing for users.

I like `describes` or `represents`.